native/macos/build.cake

```diff
@@ -47,7 +47,7 @@ Task("libHarfBuzzSharp")
     .WithCriteria(IsRunningOnMac())
     .Does(() =>
 {
-    Build("x86_64");
+    Build("x86_64", "arm64");
 
     RunLipo(OUTPUT_PATH, "libHarfBuzzSharp.dylib", new [] {
        (FilePath) "x86_64/libHarfBuzzSharp.dylib"
```

Comment:
This should be run for the arm64 arch as well if I am not mistaken

Reply:
That is correct, for both libs.